downloader.go

```
@@ -116,44 +134,94 @@ errloop:
 	log.Printf("All done, integrations installed to '%s'", *outdir)
 }
 
-// expandConfig extends defaults to integrations and performs basic validation
-func expandConfig(conf *config) error {
-	// Build template for default URL
-	globalTemplate, err := template.New("url").Parse(conf.URL)
+// expand compiles templates, extends defaults to integrations and performs basic validation
+func (conf *config) expand(useStaging, overrideLatest bool) error {
+	if useStaging {
+		conf.URL = conf.StagingUrl
```

shouldn't we validate here for a non empty value? in order to fail fast

---

Done!